pkg/ops/rawDiskGeneration.go

```diff
@@ -741,6 +737,49 @@ func (r *RawImage) copyShimOrGrub(target, which string) error {
 		return fmt.Errorf("invalid which value: %s", which)
 	}
 
+	// Alpine does not provide a shim nor a grub file, so we need to copy the shim from the docker image that we ship as fallback artifcat
+	if model, flavor, err := r.GetModelAndFlavor(); flavor == "alpine" && model != "generic" && err == nil {
+		if arch == "arm64" {
+			// Copy the grub in the shim place
+			err = utils.CopyFile(
+				r.config.Fs,
+				filepath.Join("/efi", constants.EfiBootPath, "grub.efi"),
+				filepath.Join(target, constants.EfiBootPath, "bootaa64.efi"),
+			)
+			if err != nil {
+				return fmt.Errorf("could not write file %s at dir %s from %s", "bootaa64.efi", target, filepath.Join("/efi", constants.EfiBootPath, "grub.efi"))
+			}
+			// Also copy it into the grub name to have the same files as in other flavors
+			err = utils.CopyFile(
+				r.config.Fs,
+				filepath.Join("/efi", constants.EfiBootPath, "grub.efi"),
+				filepath.Join(target, constants.EfiBootPath, "grub.efi"),
+			)
+			if err != nil {
+				return fmt.Errorf("could not write file %s at dir %s from %s", "grub.efi", target, filepath.Join("/efi", constants.EfiBootPath, "grub.efi"))
```

Comment:
better store this in a variable to avoid repetition. I mean the `filepath.Join("/efi", constants.EfiBootPath, "grub.efi"),` part. Maybe the others too (although they repeat less).